File: Javascript/Algorithms/1528.js
/**
 * @param {string} s
 * @param {number[]} indices
 * @return {string}
 */
var restoreString = function(s, indices) {
    let v = Array.from(s);
    for (i = 0; i < s.length; i++){
        v[indices[i]] = s[i];
    }
    return v.join("");
}

File: Python/Algorithms/1528.py
class Solution:
    def restoreString(self, s: str, indices: List[int]) -> str:
        # uses correct space.
        l = list(s)
        # go through and perform assignments.
        for i, j in enumerate(indices):
            l[j] = s[i]
        return "".join(l)
